apps/public-docsite-v9/src/Concepts/AdvancedConfiguration.stories.mdx

```diff
@@ -32,7 +32,41 @@ function MyComponent(props) {
 
 You can check complete example at [CodeSandbox](https://codesandbox.io/s/fluentuireact-components-render-into-iframe-l62ke).
 
-### Content Security Policies
+### Media Queries Sorting
+
+Fluent UI React v9 uses Griffel as CSS-in-JS to apply styles. To provide deterministic behavior, styles applied by Griffel [are sorted]((https://griffel.js.org/react/guides/atomic-css#lvfha-order-of-pseudo-classes), the same applies to media queries. Fluent UI does not provide an opinionated default to sort media queries as the order may vary depending on the specific needs of the application.
+
+To configure the sorting behavior, please pass `compareMediaQueries` function with the same signature as sort functions in e.g. `Array.prototype.sort`.
+
+```js
+import { createDOMRenderer } from '@griffel/react';
+
+const mediaQueryOrder = [
```

Comment:
should we mention third party comparators here too?

Reply:
Added a mention